lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -747,44 +747,51 @@ def _delete_boot_diagnostic_container(
 
     def _get_node_information(self, node: Node) -> Dict[str, str]:
         information: Dict[str, Any] = {}
-        for key, method in self._environment_information_hooks.items():
-            node.log.debug(f"detecting {key} ...")
-            try:
-                value = method(node)
-                if value:
-                    information[key] = value
-            except Exception as identifier:
-                node.log.exception(f"error on get {key}.", exc_info=identifier)
+        if node.capture_vm_information:
```

Comment:
Reduce indent levels
```suggestion
        if node.capture_vm_information is False:
            return information
```